opencog/cython/PythonEval.cc

```diff
@@ -4,6 +4,7 @@
  *         Ramin Barati <rekino@gmail.com>
  *         Keyvan Mir Mohammad Sadeghi <keyvan@opencog.org>
  *         Curtis Faith <curtis.m.faith@gmail.com>
+ *         Alexey Potapov <alexey@singularitynet.io>
```

Comment:
Umm, why isn't Alexey creating this pull req? If this code really is from him, he should be the one doing the legwork on it?

Reply:
Because I have an incentive to keep `opencog/atomspace` and `opencog/singnet` in sync.

However, that sort of work should take place on `opencog/atomspace`.